config/environments/production.rb

```diff
@@ -44,20 +43,15 @@
   config.log_level = :info
 
   # Prepend all log lines with the following tags.
-  # config.log_tags = [ :request_id ]
+  config.log_tags = [:request_id]
```

Comment:
I think they commented this out on purpose, it was commented on router-api as well